tests/cli/test_rasa_data.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+
+def test_data_split_nlu(run_in_default_project):
+    run_in_default_project(
+        "data", "split", "nlu", "--nlu", "data/nlu.md", "--training-fraction", "0.75"
+    )
+
+    assert os.path.exists("train_test_split")
+    assert os.path.exists(os.path.join("train_test_split", "test_data.md"))
+    assert os.path.exists(os.path.join("train_test_split", "training_data.md"))
+
+
+def test_data_convert_nlu(run_in_default_project):
+    run_in_default_project(
+        "data",
+        "convert",
+        "nlu",
+        "--data-file",
+        "data/nlu.md",
+        "--out-file",
+        "out_nlu_data.json",
+        "-f",
+        "json",
+    )
+
+    assert os.path.exists("out_nlu_data.json")
+
+
+def test_data_split_help(run):
```

Comment:
does it really make sense to test for the help messages?

Reply:
As the command line interface should be stable, we should not just change any arguments in the future. If someone change something on the command line interface it should be well thought out. I just want to avoid some changes by mistake lead to missing arguments.

Reply:
Makes sense :+1: